lib/semver.go

```diff
@@ -47,15 +47,15 @@ func SemVerParser(tfconstraint *string, tflist []string) (string, error) {
 		}
 	}
 
-	return "", fmt.Errorf("did not find version matching constraint: %s", *tfconstraint)
+	return "", fmt.Errorf("Did not find version matching constraint: %s", *tfconstraint)
 }
 
 // PrintInvalidTFVersion Print invalid TF version
 func PrintInvalidTFVersion() {
-	logger.Info("Version does not exist or invalid terraform version format.\n Format should be #.#.# or #.#.#-@# where # are numbers and @ are word characters.\n For example, 0.11.7 and 0.11.9-beta1 are valid versions")
+	logger.Warn("Version does not exist or invalid terraform version format.\n\tFormat should be #.#.# or #.#.#-@# where # are numbers and @ are word characters.\n\tFor example, 0.11.7 and 0.11.9-beta1 are valid versions")
```

Comment:
Should these not be errors?

Reply:
Hmm, not sure I'm getting the point.
I wanted to make these look differently instead of INFO, and WARN looked good.
Or do you mean these should better be ERR?

Reply:
Forgot this PR was merged.. maybe for another time :)

Reply:
Ah, I was reading "Should these not be errors?" as you meant INFO was okay 🤣 
Pushed to https://github.com/warrensbox/terraform-switcher/pull/549
Thanks